slack_sdk/audit_logs/v1/async_client.py

```diff
@@ -49,6 +55,7 @@ def __init__(
         user_agent_prefix: Optional[str] = None,
         user_agent_suffix: Optional[str] = None,
         logger: Optional[logging.Logger] = None,
+        retry_handlers: List[RetryHandler] = async_default_handlers,
```

Comment:
The default one consists of only the `(Async)ConnectionErrorRetryHandler` instance with its defaults.